at_secondary/at_secondary_server/config/config.yaml

```diff
@@ -132,4 +132,4 @@ sync:
 #set to true in testing mode, false by default
 #IMPORTANT NOTE : please set testingMode to true only if you know what you're doing. Set to false when not testing
 testing:
-  testingMode: false
+  testingMode: true
```

Comment:
This should always be false in config.yaml, because this file is what gets sent to prod. Should only ever be set to true by injecting into docker container env via the github workflow

Reply:
Given that this came up in another situation also, I'd like there to be a test which asserts that the config.yaml never contains "testingMode: false"